build.sh

```diff
@@ -83,9 +83,9 @@ if [ ! -d "$DIR/build/apache-rat-${rat_version}" ]; then
   tar zxf build/${filename} -C build/
 fi
 
-echo "Running RAT license check"
+echo "Running RAT license check on $DIR"
 output=$(java -jar $DIR/build/apache-rat-${rat_version}/apache-rat-${rat_version}.jar -d $DIR -E rat-excludes.txt)
-if [[ ! $(echo "$output" | grep '0 Unknown Licenses') ]]; then
+if [[ ! $(echo "$output" | grep '^0 Unknown Licenses') ]]; then
```

Comment:
Nice catch.

Reply:
I had 10 files generated in the `public` but passed the licenses check since the command `grep '0 Unknown Licenses'` matched it.
```
10 Unknown Licenses
```